kratos/includes/kernel.h

```diff
@@ -70,7 +70,7 @@ class KRATOS_API(KRATOS_CORE) Kernel {
         @see KratosApplication
 
     */
-    Kernel();
+    Kernel(bool IsDistributedRun = false);
```

Comment:
Why we need to expose this to all? It shouldn't be a protected  constructor for MPI kernel to set it correctly?

Reply:
There is no MPI-specific version of the Kernel currently. I can add it as desired, but I did not do it at the moment, since the only difference between the two would be setting a static member variable to a different value.